lara-typescript/src/section-authoring/components/item-edit-dialog.tsx

```diff
@@ -156,6 +156,7 @@ export const ItemEditDialog: React.FC<IItemEditDialogProps> = ({
         return <ManagedInteractiveAuthoring
                 managedInteractive={managedInteractive}
                 libraryInteractive={libraryInteractive}
+                interactive_item_id={`interactive_${itemToEdit.id}`}
```

Comment:
looking at this again, I suggest making a method like 
``` 
const interactiveFromItemToEdit = (itemToEdit: ISectionItem) => {
  const interactive = camelToSnakeCaseKeys(itemToEdit.data);
  interactive.interactive_item_id = `interactive_${itemToEdit.id}`;
  return interactive;
};
```

And then use that to make the `managedInteractive` here and the `interactive` below like:
`const managedInteractive = interactiveFromItemToEdit(itemToEdit);`

With that change you don't need to add the new property to the ManageInteractiveAuthoring and MwInteractiveAuthoring components. 
And the existing code which was setting the `interactive_item_id` like on line 114 of the old managed-interactives/index.tsx  could remain the same.

Reply:
Nice. That's definitely better. I've made those changes in the latest commit.